src/workspacer/workspacer.csproj

```diff
@@ -29,10 +29,16 @@
     <ProjectReference Include="..\workspacer.FocusIndicator\workspacer.FocusIndicator.csproj" ReferenceOutputAssembly="false" />
     <ProjectReference Include="..\workspacer.TitleBar\workspacer.TitleBar.csproj" ReferenceOutputAssembly="false" />
     <ProjectReference Include="..\workspacer.Watcher\workspacer.Watcher.csproj" ReferenceOutputAssembly="false" />
+	  <ProjectReference Include="..\workspacer.FocusBorder\workspacer.FocusBorder.csproj" ReferenceOutputAssembly="false" />
   </ItemGroup>
   <Target Name="CopyPluginsAfterBuild" AfterTargets="Build">
     <Copy SourceFiles="$(SolutionDir)src\workspacer.Bar\$(OutDir)workspacer.Bar.dll" DestinationFolder="$(TargetDir)plugins\workspacer.Bar\" />
     <Copy SourceFiles="$(SolutionDir)src\workspacer.Gap\$(OutDir)workspacer.Gap.dll" DestinationFolder="$(TargetDir)plugins\workspacer.Gap\" />
+	<Copy SourceFiles="$(SolutionDir)src\workspacer.FocusBorder\$(OutDir)workspacer.FocusBorder.dll" DestinationFolder="$(TargetDir)plugins\workspacer.FocusBorder\" />
```

Comment:
Weird indentation here with a mixture of tabs and spaces. Keep to spaces only please :)